packages/web-components/src/label/label.ts

```diff
@@ -0,0 +1,40 @@
+import { attr, FASTElement } from '@microsoft/fast-element';
+import { LabelSize, LabelWeight } from './label.options.js';
+// import { LabelSize, LabelWeight } from "./label.options.js";
+/**
+ * The base class used for constructing a fluent-label custom element
+ * @public
+ */
+export class Label extends FASTElement {
+  /**
+   * 	Specifies styles for label when associated input is a required field
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: required
+   */
+  @attr({ mode: 'boolean' })
+  public required: boolean = false;
+
+  /**
+   * 	Specifies font size of a label
+   *
+   * @public
+   * @default 'medium'
+   * @remarks
+   * HTML Attribute: size
+   */
+  @attr
+  public size: LabelSize = LabelSize.medium;
```

Comment:
Reminder, set attribute defaults for purely visual attributes in CSS not in the class.